web/README.md

```diff
@@ -6,3 +6,16 @@ Go to the `generate_site/` directory,
 install the requirements from `requirements.txt`,
 and that wil generate the `index.html` and `about.html`
 files for the web.
+
+## Adding Opengraph Meta tag
+
+When adding a new page to the site, please be sure to update the OpenGraph
+meta tag. These tags are useful when sharing the links on social media,
+it generates a nice preview of the site.
+
+To add these info, see the examples of the og_ meta tags in generate.py ``templates``
+variable where we added ``og_title``, etc. to the ``about`` and ``index`` pages.
```

Comment:
```suggestion
To add this info, refer to the `og_meta` tags defined for the current templates (about, index) in `generate.py`.
```
 
I found the previous wording a bit confusing, so this is more of an optional suggestion than a hard-needed change.